test/lib/sun/hotspot/WhiteBox.java

```diff
@@ -318,6 +318,36 @@ public         int     getMethodCompilationLevel(Executable method, boolean isOs
     Objects.requireNonNull(method);
     return getMethodCompilationLevel0(method, isOsr);
   }
+  public         int     getMethodDecompileCount(Executable method) {
```

Comment:
As this class is marked as `@Deprecated`, do we need to add the methods here as well?

Reply:
Unfortunately yes :(
I first didn't but there are still tests using it and they'll get warning otherwise. To make matters worse, some of them parse the output, so I gave up and added the methods to `sun/hotspot/WhiteBox.java` as well.